internal/pkg/cli/deploy/env.go

```diff
@@ -293,6 +293,17 @@ func (d *envDeployer) cidrPrefixLists(in *DeployEnvironmentInput) ([]string, err
 	return cidrPrefixListIDs, nil
 }
 
+func (d *envDeployer) publicALBSourceIPList(in *DeployEnvironmentInput) []string {
+	if in.Manifest == nil || len(in.Manifest.GetPublicALBSourceIPs()) == 0 {
+		return nil
+	}
+	var publicALBSourceIPList []string
+	for _, sourceIP := range in.Manifest.GetPublicALBSourceIPs() {
+		publicALBSourceIPList = append(publicALBSourceIPList, string(sourceIP))
+	}
+	return publicALBSourceIPList
```

Comment:
nit:
```suggestion
	ips := make([]string, len(in.Manifest.GetPublicALBSourceIPs()))
	for i, sourceIP := range in.Manifest.GetPublicALBSourceIPs() {
		ips[i] = string(sourceIP)
	}
	return ips
```
I follow [this rule from the naming guideline](https://a-nickels-worth.blogspot.com/2016/04/a-guide-to-naming-variables.html#orgheadline12) for a past sr. principal. In this case `ips` carry the same amount of info as `publicALBSourceIPList` because it's in the field of vision. Might as well make it shorter.